1/noisifier.py

```diff
@@ -0,0 +1,76 @@
+# Author: Mikita Sazanovich
+
+import os
```

Comment:
It is optional but consider trying sometime to use Pathlib as a modern alternative

Reply:
Did not know about it. But it is really convenient as I may say.